catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -359,6 +365,41 @@ private QueryRequest getRequestWithTagsFilter(QueryRequest request) {
         return newRequest;
     }
 
+    public QueryRequest getRequestWithValidityFilter(QueryRequest input) {
```

Comment:
This is the method I was talking about when I mentioned the testability of this code. I guess I forgot it was public. But like I said there, yea it should be private.